CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+### 0.0.7 - 19.12.2024
```

Comment:
Trenger vi denne? 🤓 Brukes den til noe?

Reply:
Tenker det er nyttig for å gå tilbake å se hva som har skjedd i forkjellige versjoner når man opplever bugs feks, lettere enn å lete i commits

Reply:
Og for brukerene slik at de kan se hvorfor de vil oppdatere til forskjellige versjoner